src/algorithms/tracking/gnuradio_blocks/dll_pll_veml_tracking.cc

```diff
@@ -652,6 +654,7 @@ void dll_pll_veml_tracking::start_tracking()
     Signal_[0] = d_acquisition_gnss_synchro->Signal[0];
     Signal_[1] = d_acquisition_gnss_synchro->Signal[1];
     Signal_[2] = d_acquisition_gnss_synchro->Signal[2];
+    d_extend_correlation_symbols = d_trk_parameters.extend_correlation_symbols;
```

Comment:
Duplicated?

Reply:
No. That's reinitialization to default value in case, this channel was tracking GEO satellite, lost lock and got switched to MEO/IGSO satellite. It can go to `else` branch https://github.com/gnss-sdr/gnss-sdr/blob/1043b02e28efdc510b8308ed82c1cbfd5599ebc9/src/algorithms/tracking/gnuradio_blocks/dll_pll_veml_tracking.cc#L775-L778, but I decided to not write duplicate code.